python-frontend/src/main/java/org/sonar/plugins/python/api/LocationInFile.java

```diff
@@ -53,4 +64,28 @@ public int endLine() {
   public int endLineOffset() {
     return endLineOffset;
   }
+
+
+  public DescriptorsProtos.LocationInFile toProtobuf() {
+    return DescriptorsProtos.LocationInFile.newBuilder()
+      .setFileId(fileId)
+      .setStartLine(startLine)
+      .setStartLineOffset(startLineOffset)
+      .setEndLine(endLine)
+      .setEndLineOffset(endLineOffset)
+      .build();
+  }
```

Comment:
_subjective_ In the other analyzers, we prefer protobuf-related stuff to be in a separate class, I think it would make sense do it here as well, to avoid having as much boilerplate in the classes themselves.